apps/webapp/app/v3/marqs/sharedQueueConsumer.server.ts

```
@@ -93,23 +120,48 @@ export type SharedQueueConsumerOptions = {
   private _enabled = false;
   private _options: Required<SharedQueueConsumerOptions>;
   private _perTraceCountdown: number | undefined;
-  private _lastNewTrace: Date | undefined;
+  private _traceStartedAt: Date | undefined;
   private _currentSpanContext: Context | undefined;
-  private _taskFailures: number = 0;
-  private _taskSuccesses: number = 0;
+  private _reasonStats: Record<string, number> = {};
   private _currentSpan: Span | undefined;
   private _endSpanInNextIteration = false;
   private _tasks = sharedQueueTasks;
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)